org.d3cod3.Mosaic.yml

```diff
@@ -0,0 +1,449 @@
+app-id: org.d3cod3.Mosaic
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: mosaic
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  - --socket=fallback-x11
+  
+  - --socket=pulseaudio
+  # Needs access to webcams and so
+  - --device=dri
+  - --device=all
```

Comment:
with `--all` you don't need `--device=dri`

Reply:
good catch!